src/proxy.rs

```diff
@@ -28,9 +31,9 @@ use {into_url, IntoUrl, Url};
 /// check each `Proxy` in the order it was added. This could mean that a
 /// `Proxy` added first with eager intercept rules, such as `Proxy::all`,
 /// would prevent a `Proxy` later in the list from ever working, so take care.
-#[derive(Clone, Debug)]
+#[derive(Debug)]
 pub struct Proxy {
-    intercept: Intercept,
+    pub(crate) inner: HyperProxy<()>,
```

Comment:
I always have to double check, but I *think* `pub(path)` requires a newer Rust compiler version than the minimum supported...

Reply:
Oh ok. Is there an official minimum version to support?

Reply:
Oh what do know, it turns out there isn't a specific version set in the `.travis.yml`... I just assumed there was of 1.17. This may not be an issue, then...